docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc

```diff
@@ -56,6 +56,31 @@ Fleet::
 * Improves default settings for {fleet} component templates {kib-pull}114101[#114101]
 * Report `installing` status while package install is in progress {kib-pull}111875[#111875]
 
+[discrete]
+[[known-issues-7.16.x]]
+=== Known issues
+
+{kib-pull}121059[#121059]
```

Comment:
You should either use the structure from the template (if we expect to add more known issues), or remove the collapsible section. If you keep the collapsible section, it should start with this:

```
[[known-issue-issue 121059]]
.Add short description here
[%collapsible]
====
```